File: app/src/main/java/com/example/newyorktimearticles/data/model/Result.java
package com.example.newyorktimearticles.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Result {
    @SerializedName("results")
    @Expose
    private List<Articles> articlesList;

    public List<Articles> getArticlesList() {
        return articlesList;
    }
}

File: app/src/main/java/com/gemidroid/newyorktimearticles/data/model/Result.kt
package com.gemidroid.newyorktimearticles.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {
    @SerializedName("results")
    @Expose
    val articlesList: List<Articles>? = null
}